README.md

```diff
@@ -23,11 +27,10 @@ your client's information.
   * To find out more information about FABook, please take a look at our [User Guide](docs/UserGuide.md)!
 
 ## Developer Guide:
-  * To find out more technical information or if you want to help improve FABook, please take a look at our
-  * [Developer Guide](docs/DeveloperGuide.md)!
+  * To find out more technical information or if you want to help us improve your experience with FABook, please take a look at our [Developer Guide](docs/DeveloperGuide.md)!
```

Comment:
Can link to here instead, https://ay2223s1-cs2103t-t10-2.github.io/tp/DeveloperGuide.html